simulation/src/test/resources/report.md

```diff
@@ -644,6 +692,36 @@ client=9 endpoint	client_mean=PT0.000001S
 <tr><td><image width=400 src="https://media.githubusercontent.com/media/palantir/dialogue/develop/simulation/src/test/resources/server_side_rate_limits[UNLIMITED_ROUND_ROBIN].png" /></td><td><image width=400 src="server_side_rate_limits[UNLIMITED_ROUND_ROBIN].png" /></td></tr></table>
 
 
+## `server_side_rate_limits_with_sticky2_clients_fairness_across_multiple_clients[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR]`
```

Comment:
I think I'd prefer if we updated the existing test rather than adding a new one, that will be a better comparison, and we don't intend to invest in the original implementation moving forward.

Reply:
Ack

Reply:
Should be done.